src/main/scala/it/pagopa/interop/attributeregistryprocess/service/CatalogManagementService.scala

```diff
@@ -0,0 +1,10 @@
+package it.pagopa.interop.attributeregistryprocess.service
```

Comment:
[same](https://github.com/pagopa/interop-be-attribute-registry-process/pull/1/files#r1135229414)